diff -up sinthgunt-2.0.3/share/sinthgunt.desktop.ori sinthgunt-2.0.3/share/sinthgunt.desktop
--- sinthgunt-2.0.3/share/sinthgunt.desktop.ori	2012-02-25 08:00:02.231191879 -0500
+++ sinthgunt-2.0.3/share/sinthgunt.desktop	2012-02-25 08:00:08.668191439 -0500
@@ -3,7 +3,7 @@ Name=sinthgunt
 GenericName=Video converter
 Comment=Convert video files
 Exec=sinthgunt
-Icon=sinthgunt.png
+Icon=sinthgunt
 Type=Application
 Terminal=false
 Encoding=UTF-8
